lib/Common/DataStructures/List.h

```diff
@@ -360,6 +360,7 @@ namespace JsUtil
         void SetItem(int index, const T& item)
         {
             EnsureArray(index + 1);
+            // TODO: (SWB)(leish) find a way to force user defined copy constructor
```

Comment:
I implemented StackFrame& operator= for this, I meant if we have a way to force user defined copy constructor call, like 
this->buffer[index] = T::T(item);

it will be good, so we don't implement operator = for T. because the default operator = does not call write barrier on each field


---
In reply to: [92253745](https://github.com/Microsoft/ChakraCore/pull/2191#discussion_r92253745) [](ancestors = 92253745)